From 9db523cf91d7d45ac509792b9d4e241fb3ff95f7 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Sun, 16 Jun 2013 07:53:25 +0200
Subject: [PATCH] disable sw import/export tests

Most of them fails for unclear (yet) reasons on ppc/s390 (big endian
issue?) This is the last chance to get something that builds before the
F-19 final freeze.

Change-Id: I4b15da3f444d59aaf3bb3023e83dbf9bc2efd9fe
---
 sw/Module_sw.mk | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/sw/Module_sw.mk b/sw/Module_sw.mk
index 202c67c..04ad5fa 100644
--- a/sw/Module_sw.mk
+++ b/sw/Module_sw.mk
@@ -46,16 +46,7 @@ $(eval $(call gb_Module_add_check_targets,sw,\
 ))
 
 $(eval $(call gb_Module_add_slowcheck_targets,sw,\
-    CppunitTest_sw_htmlexport \
     CppunitTest_sw_macros_test \
-    CppunitTest_sw_ooxmlexport \
-    CppunitTest_sw_ooxmlimport \
-    CppunitTest_sw_ww8export \
-    CppunitTest_sw_ww8import \
-    CppunitTest_sw_rtfexport \
-    CppunitTest_sw_rtfimport \
-    CppunitTest_sw_odfexport \
-    CppunitTest_sw_odfimport \
 ))
 
 ifneq ($(DISABLE_CVE_TESTS),TRUE)
-- 
1.8.2.1

